src/mmw/apps/modeling/mapshed/calcs.py

```diff
@@ -203,6 +205,38 @@ def manure_spread(aeu):
 
     return n_list, p_list
 
+def ls_factors(lu_strms, areas, avg_slope):
+    results = [0.0] * len(lu_strms)
+    for i in range(len(lu_strms)):
+        results[i] = ls_factor(lu_strms[i] * KM_PER_M, areas[i], avg_slope)
+    return results
+
+def ls_factor(stream_length, area, avg_slope):
+    # units --> stream_length:km, area:hectare, avg_slope:%
+    slope_cell_size = 30 #for posterity
+    if 0 <= avg_slope <= 1.0:
+        m = 0.2
```

Comment:
Calculating `m` inside the loop implies that it is different in each iteration, which is not the case. I think it would help clarity to move it out.

For `s` and `t`, it may be better to use the numbers directly, and add the explanation in a comment. This should reduce the number of variables to keep track of, as well as be more in sync with the original where they did use straight numbers instead of these variables.